web/app/themes/ppj/src/vue/FindAJob.vue

```diff
@@ -120,14 +119,12 @@
               :key="index"
               v-on:click="handleVacancyClick(job.jobLocationGroupId)">
             <job-summary :distance="job.distance"
-                         :distance-time="job.distanceTime"
                          :position="job.role"
```

Comment:
There are still unused attributes in here which are no longer valid.

- Remove `position` and `prison-page-link` attributes.